contracts/pcv/utils/ERC20Allocator.sol

```diff
@@ -171,7 +171,7 @@ contract ERC20Allocator is IERC20Allocator, CoreRef, RateLimitedV2 {
         address token,
         address to,
         uint256 amount
-    ) external onlyGovernor {
+    ) external override onlyGovernor {
```

Comment:
we aren't changing the access controls on the allocator. We just need to override the sweep function in CoreRefV2 because the same function is defined twice. Alternatively, we could just remove the sweep function in ERC20Allocator because the CoreRefV2 contract has it and ERC20Allocator inherits CoreRefV2.

<img width="758" alt="Screen Shot 2022-10-31 at 1 44 32 PM" src="https://user-images.githubusercontent.com/34463580/199106913-89e8c76d-2bd8-4b4d-84c2-819bd7de0149.png">